packages/runtime/test/transport/relationships.test.ts

```diff
@@ -744,7 +748,7 @@ describe("RelationshipDecomposition", () => {
         templateId2 = relationship2.template.id;
 
         await createRelationshipData(services1, services3);
-        multipleRecipientsMessageId = (await sendMessageToMultipleRecipients(services1.transport, [services2.address, services3.address])).id;
+        multipleRecipientsMessageId = await sendMessageToMultipleRecipients(services1.transport, [services2.address, services3.address]);
```

Comment:
There was no need no remove the id (and to type `multipleRecipientsMessageId` to any)